samcli/hook_packages/terraform/hooks/prepare/translate.py

```diff
@@ -208,44 +208,20 @@ def translate_to_cfn(tf_json: dict, output_directory_path: str, terraform_applic
             # Add resource to cfn dict
             cfn_dict["Resources"][logical_id] = translated_resource
 
-            if resource_type == TF_AWS_LAMBDA_LAYER_VERSION:
```

Comment:
❤️ on this refactor. I love it when conditional branching with `if` reduces and control flow 😺 is more direct.